### ISSUE TITLE
Application is shutting down without any notice when opening a view and refreshing by pulling down

### ISSUE THREAD

**Issue by chosn** [LeoKlaus/Paperparrot#114]
**Describe the bug**
When choosing a saved view on the top-left corner (I called it "Recent"), refreshing the displayed content results in application to crash.
The "Recent"-view fiter setting is:
- !(Tag Inbox)
- Added: "last 7 days"

**To Reproduce**
1. Start Paperparrot
2. Choose view Recent via top left corner
3. Pull down to refresh
4. During "fetching documents from server" application closes without any further notification

**Expected behavior**
Application must not close but display the document list

**Screenshots**
none

**Environment(please complete the following information):**
 - Device: iPhone SE (3rd Gen)
 - OS: iOS 16.6
 - Version 20G75
 - Paperparrot version 1.6.0
 - Server Version: Paperless-ngx 1.17.0

**Comment by LeoKlaus** [comment]
Hi chosn,

thank you for the bug report! I don't quite understand what you mean by !(Tag Inbox).
I assume what you mean is that under tags you've selected the tag "Inbox" to be excluded?

It would be best if you could post the saved view as it is returned by the api, as that is what Paperparrot parses.

To get there, head to http(s)://your-paperless-instance:port/api/saved_views/ and look for the saved view in question.

It should look something like this:
``` JSON
{
            "id": 3,
            "name": "Neu",
            "show_on_dashboard": true,
            "show_in_sidebar": false,
            "sort_field": "created",
            "sort_reverse": true,
            "filter_rules": [
                {
                    "rule_type": 6,
                    "value": "7"
                }
            ],
            "owner": {
                "id": 2,
                ...
            },
            "user_can_change": true
        }
```

You should omit the "owner"-part as that may contain information about you. The interesting part for me is:
sort_field
sort_reverse
filter_rules

**Comment by chosn** [comment]
Hi LeoKlaus.
Your assumption is correct. 👍 
I exclude the tag "Inbox".
``` JSON
{
            "id": 3,
            "name": "Recent",
            "show_on_dashboard": true,
            "show_in_sidebar": true,
            "sort_field": "created",
            "sort_reverse": true,
            "filter_rules": [
                {
                    "rule_type": 17,
                    "value": "1"
                },
                {
                    "rule_type": 20,
                    "value": "added:[-1 week to now]"
                }
            ],
            "owner": {
                "id": 4,
                ....
            },
            "user_can_change": true
        }
```

**Comment by LeoKlaus** [comment]
Great, thank you very much. I assume you've closed the issue by accident, so I'll reopen it.

Update: I can reproduce the issue on my device, now I'll just have to find out what causes this.

**Comment by LeoKlaus** [comment]
I've fixed this with update 1.7.0. I plan to release this update to TestFlight this week and the AppStore next week.
